### PR TITLE
Chapter 11

### PR DESCRIPTION
**Pull request by mohdzakwan99**
ready

### CODE REVIEW

**Comment by mohdzakwan99** [comment]
ready

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/static_pages.scss:1]: please delete this file

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/users.scss:2]: please delete this file

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -59,9 +53,17 @@ def destroy
 
   private
 
+  def correct_user
+    @user = User.find(params[:id])
+    return if @user==current_user?(@user)
+    flash[:danger]="You are not authorized!"
```
Comment (anlvq-1058) [inline app/controllers/users_controller.rb:59]:
```suggestion
    flash[:danger] = "You are not authorized!"
```

File: package.json [hunk 4]
```diff
@@ -6,8 +6,11 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.4.2",
+<<<<<<< HEAD
     "bootstrap": "3.4.1",
     "jquery": "3.4.1",
+=======
+>>>>>>> 6ce05c1... chapter 6
```
Comment (anlvq-1058) [inline package.json:13]: please fix this conflict code

File: test/controllers/users_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/users_controller_test.rb:1]: delete this file, please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by mohdzakwan99** [comment]
ready

**Comment by mohdzakwan99** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
small comment

**Comment by mohdzakwan99** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**